src/main/java/org/dwcj/controls/htmlcontainer/sinks/HtmlContainerPageLoadedEventSink.java

```diff
@@ -7,16 +7,19 @@
 import org.dwcj.controls.htmlcontainer.HtmlContainer;
 import org.dwcj.controls.htmlcontainer.events.HtmlContainerPageLoadedEvent;
 
+import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.function.Consumer;
 
 public final class HtmlContainerPageLoadedEventSink {
 
-    private final Consumer<HtmlContainerPageLoadedEvent> target;
+    private final ArrayList<Consumer<HtmlContainerPageLoadedEvent>> targets;
     private final HtmlContainer container;
 
     @SuppressWarnings({"static-access"})
```

Comment:
* Don't suppress warnings
* Use ` com.basis.bbj.proxyif.SysGuiEventConstants.ON_PAGE_LOADED`  instead

Do the same in `HtmlContainerNativeJavascriptEventSink`